src/pymovements/gaze/experiment.py

```diff
@@ -31,55 +31,47 @@
 class Experiment:
     """Experiment class for holding experiment properties.
 
-    Attributes
```

Comment:
is pydoclint complaining about this section? If we delete this, we won't have documented anymore that we can get the screen of the experiment by using Experiment.screen.

Reply:
I'll double check